extensions/amp-story-dev-tools/0.1/amp-story-dev-tools.css

```diff
@@ -18,10 +18,85 @@ amp-story-dev-tools {
   width: 100% !important;
   height: 100% !important;
   box-sizing: border-box !important;
-  background-color: #121212 !important;
+  background-color: black !important;
   font-size: 16px !important;
   font-family: 'Poppins', sans-serif !important;
   color: white !important;
   padding: 30px 10% !important;
   text-align: center !important;
 }
+
+.i-amphtml-story-dev-tools-container {
+  height: 100% !important;
+}
+
+.i-amphtml-story-dev-tools-header {
+  display: flex !important;
+  align-items: center !important;
+  justify-content: space-between !important;
+  color: white !important;
+  height: 40px !important;
+  box-sizing: border-box !important;
+}
+
+.i-amphtml-story-dev-tools-brand {
+  display: inline-flex !important;
+}
+
+.i-amphtml-story-dev-tools-brand-text > span {
+  display: block !important;
+  letter-spacing: 0.1em !important;
+  line-height: 1.2em !important;
+  padding-left: 10px !important;
+}
+
+.i-amphtml-story-dev-tools-brand-text > span:nth-child(2) {
+  color: #79B3FF !important;
+  font-size: 10px !important;
+  text-align: left;
```

Comment:
I see `text-align: center` on `.amp-story-dev-tools` and it's being over-ridden here.
Removing `text-align: center` and `text-align: left` seems to have no visible effect.
We might want to remove both lines until they are more useful in the layout.